FiveCalls/FiveCalls/LocationHeader.swift

```diff
@@ -90,9 +99,17 @@ struct LocationHeader: View {
 struct LocationHeader_Previews: PreviewProvider {
     static var previews: some View {
         VStack {
-            LocationHeader(location: nil, fetchingContacts: true)
-            LocationHeader(location: nil, fetchingContacts: false)
-            LocationHeader(location: UserLocation(address: "19444"), fetchingContacts: false)
+            LocationHeader(location: nil, isSplit: false, fetchingContacts: true)
+            LocationHeader(location: nil, isSplit: true, fetchingContacts: false)
+            LocationHeader(
+                location: UserLocation(address: "19444"),
```

Comment:
i don't know what this preview thing is about -- is this just for development? seems weird to hard-code here as zip codes can be redistricted.

Reply:
Yep, these are a set of view configurations that will show in Xcode so you can see how the different settings are rendered, development only. They don't even depend on the zip in question being split because at this point down the view hierarchy we're just describing how to render so we declare "this should be shown as split" or not